tests/test.cpp

```diff
@@ -150,8 +150,9 @@ flatbuffers::unique_ptr_t CreateFlatBufferTest(std::string &buffer) {
 void AccessFlatBufferTest(const uint8_t *flatbuf, size_t length) {
 
   // First, verify the buffers integrity (optional)
-  flatbuffers::Verifier verifier(flatbuf, length);
+  flatbuffers::Verifier verifier(flatbuf, length + 123);
```

Comment:
Ah, I add it by mistake to this patch :D, anyway I'll update it as you suggested.